
std_msgs/Header header
uint8 xloc_state
string xloc_state_message
string current_active_map
float64 reliability
float64 matching_score
